App/screens/CredentialOffer.tsx

```diff
@@ -98,7 +132,7 @@ const CredentialOffer: React.FC<Props> = ({ navigation, route }) => {
   return (
     <View style={styles.container}>
       <ModularView
-        title={parseSchema(credential?.metadata.schemaId)}
+        title={parseSchema((credential?.metadata as Metadata & { schemaId: string })?.schemaId)}
```

Comment:
I think you need to use .get() for metadata. @blu3beri?

Reply:
Is this the correct way to use the getter?

```
credential?.metadata.get<Record<string, string>>('_internal/indyCredential')?.['schemaId']
```